tensorboard/webapp/webapp_data_source/tb_feature_flag_data_source_types.ts

```diff
@@ -45,3 +45,5 @@ export const ENABLE_COLOR_GROUP_BY_REGEX_QUERY_PARAM_KEY =
 export const ENABLE_DARK_MODE_QUERY_PARAM_KEY = 'darkMode';
 
 export const ENABLE_LINK_TIME_PARAM_KEY = 'enableLinkTime';
+
+export const ENABLE_TIME_NAMESPACED_STATE = 'enableTimespace';
```

Comment:
Could we change it to `enableTimeNamespaced` to be consistent with other names?

Reply:
Whoops, I did this locally but forgot to push here before merging. I will send out a new PR soon to fix this.